serializer-minecraft/src/main/java/team/unnamed/creative/serialize/minecraft/MinecraftResourcePackReader.java

```diff
@@ -60,4 +61,11 @@ static MinecraftResourcePackReader minecraft() {
         return MinecraftResourcePackReaderImpl.INSTANCE;
     }
 
+    static MinecraftResourcePackReader minecraft(SerializerBlockState serializerBlockState, SerializerFont serializerFont, SerializerLanguage serializerLanguage, SerializerMetadata serializerMetadata, SerializerModel serializerModel, SerializerSoundRegistry serializerSoundRegistry) {
```

Comment:
I don't think that passing all serializers through the constructor is a good idea, we would have to modify it every time a new element

I think passing only the DeserializationErrorHandler is enough

Reply:
this is needed in case I want to throw my serializers

ideally, they would be put into interfaces

Reply:
I think it should be implemented later, with #23